.travis.yml

```diff
@@ -95,8 +95,8 @@ script:
   find setup.py testgeomapfish/*.py testgeomapfish/views -name \*.py | xargs travis/quote
 - travis/run-on.sh /tmp/travis/testgeomapfish/ docker-compose up -d
 - travis/waitwsgi http://localhost:8080/
-- travis/test-new-project http://localhost:8080/check_collector
-- travis/test-new-project http://localhost:8080/check_collector?type=all
+- travis/test-new-project http://localhost:8080/c2c/health_check
+- travis/test-new-project http://localhost:8080/c2c/health_check?max_level=100
```

Comment:
From where does this c2c comes from?

Reply:
from c2c.base_path in the production.ini

Reply:
ok